src/client/browser/index.js

```diff
@@ -0,0 +1,45 @@
+import COMMAND from '../../browser-connection/command';
+import hammerheadApi from './deps/hammerhead';
+
+
+const HEARTBEAT_INTERVAL = 30 * 1000;
+
+
+//Utils
+var XMLHttpRequest = hammerheadApi ? hammerheadApi.NativeMethods.XMLHttpRequest : window.XMLHttpRequest;
```

Comment:
Can we avoid doing it, e.g. by passing XMLHttpRequest ctor to methods (with appropriate comment which clarifies why we do this)? I don't like hammerhead dep here as well.